app/controllers/qa_controller.rb

```diff
@@ -64,13 +64,16 @@ def breadcrumbs
   def current_facet
     current_facet = facets[page - 1]
     current_facet_config = qa_config["pages"][current_facet["key"]]
-    {
+    config = {
       "question" => current_facet_config["question"],
       "description" => current_facet_config["description"],
       "hint_title" => current_facet_config["hint_title"],
       "hint_text" => current_facet_config["hint_text"],
       "facet" => current_facet
     }
+    config.merge("custom_options" => current_facet_config["custom_options"]) if current_facet_config["custom_options"].present?
```

Comment:
Sorry, my mistake, the `config.merge` needs an `!` so that the original `config` variable is updated.

The line should be: 
```
config.merge!("custom_options" => current_facet_config["custom_options"]) if current_facet_config["custom_options"].present?
``